Packs/CTFCloud/Dashboards/dashboard-CTF_Cloud_Dashboard.json

```diff
@@ -0,0 +1,451 @@
+                    "all"
+                ],
+                "definitionId": "",
+                "vcShouldIgnore": false,
+                "vcShouldKeepItemLegacyProdMachine": false,
+                "commitMessage": "",
+                "shouldCommit": false,
+                "Cache": null,
+                "name": "Fastest Top 10 users finished - Basic",
+                "prevName": "Top 10 users finished",
+                "dataType": "incidents",
+                "widgetType": "column",
+                "query": "ctfsla.runStatus:ended playbook:\"CTF - Analyst View - IAM\"",
+                "isPredefined": false,
+                "dateRange": {
+                    "fromDate": "0001-01-01T00:00:00Z",
+                    "toDate": "0001-01-01T00:00:00Z",
+                    "period": {
+                        "by": "",
+                        "byTo": "",
+                        "byFrom": "days",
+                        "toValue": null,
+                        "fromValue": 7,
+                        "field": ""
+                    },
+                    "fromDateLicense": "0001-01-01T00:00:00Z"
+                },
+                "params": {
+                    "groupBy": [
+                        "owner"
+                    ],
+                    "keys": [
+                        "avg|ctfsla.totalDuration"
+                    ],
+                    "limitType": "bottom",
+                    "showGraphValues": false,
+                    "valuesFormat": "hours",
+                    "yAxisLabel": ""
+                },
+                "category": ""
+            },
+            "reflectDimensions": true
+        },
+        {
+            "id": "f0717f30-2970-11ee-960d-4d66530a43c1",
+            "forceRange": false,
+            "x": 0,
+            "y": 2,
+            "i": "f0717f30-2970-11ee-960d-4d66530a43c1",
+            "w": 12,
+            "h": 3,
+            "widget": {
+                "id": "6fabb894-5e21-443f-872d-6c69d7b10bd3",
+                "version": 1,
+                "cacheVersn": 0,
+                "modified": "2023-07-23T15:52:32.540751271Z",
+                "packID": "",
+                "packName": "",
+                "itemVersion": "",
+                "fromServerVersion": "",
+                "toServerVersion": "",
+                "propagationLabels": [
+                    "all"
+                ],
+                "definitionId": "",
+                "vcShouldIgnore": false,
+                "vcShouldKeepItemLegacyProdMachine": false,
+                "commitMessage": "",
+                "shouldCommit": false,
+                "sort": [
+                    {
+                        "field": "id",
+                        "fieldType": "",
+                        "asc": false
+                    }
+                ],
+                "Cache": null,
+                "name": "Finished CTF Incidents",
+                "prevName": "List of all CTF incidents",
```

Comment:
Maybe should be capitalized?
List of All CTF Incidents

Reply:
No need, that is the previous name -> name is the right field to look at it.